### PR TITLE
Added section on how data/samples are processed

### PR DESCRIPTION
**Pull request by wasade**

### CODE REVIEW

File: www/americangut/FAQ.psp [hunk 1]
```diff
@@ -300,6 +301,15 @@
...
+							<p>The specifics on how the American Gut sequence data are processed can be found <a href="http://nbviewer.ipython.org/github/biocore/American-Gut/blob/master/ipynb/module2_v1.0.ipynb">here</a>, and are written up in an executable <a href="http://ipython.org/notebook">IPython Notebook</a>, which provides all the relevant processing steps in an open-source format. Be warned, processing the full American Gut dataset takes over 5,000 CPU hours right now (i.e. if you do it on your laptop it might take 7 months, even if you don’t run out of memory: this might put the time it takes to get your results in perspective). This is the processing pipeline that we use on your data. As this project is a work in progress, we are versioning the processing pipeline as there will continue to be improvements to the process as the project moves forward.</p>
+                        </div>
```
Comment (teravest) [inline www/americangut/FAQ.psp:467]: Can you make the lines shorter so we make meaningful comments?

**Comment by wasade** [comment]
@teravest, lines are broken up now. I need to add in the links @rob-knight recommended as well, so there will be a few more commits coming in

**Comment by teravest** [comment]
I'll wait and review everything at once, but I will get to it today.

File: www/americangut/FAQ.psp [hunk 2]
```diff
@@ -300,6 +301,170 @@
...
+                            is believed to be common to all Bacteria and
+                            Archaea. This gene, the 16S ribosomal RNA gene is
+                            like a barcode you find on your groceries, and
+                            serves as a marker for different organisms. There
+                            are quite a few different ways to assess the types
+                            of Bacteria and Archaea in a sample, including a
+                            variety of techniques even to look at this single
+                            gene. Every method has its biases, and comparing
+                            data between different methods is <a
+                                href="http://www.ncbi.nlm.nih.gov/pubmed/23861384">non-trivial</a>
```
Comment (teravest) [inline www/americangut/FAQ.psp:320]: This is fine the way it is, but other stuff is cited at the end of the sentence instead of in the middle.

**Comment by teravest** [comment]
I tested all the links and they work.  @ElDeveloper can you deploy this one when you are happy with it because @adamrp and I are in a meeting all afternoon.

**Comment by wasade** [comment]
Thanks @teravest!

File: www/americangut/FAQ.psp [hunk 3]
```diff
@@ -75,6 +75,7 @@
                         <li><a href="#faq22">How do I convert grams of macronutrients into percentage of calories?</a></li>
                         <li><a href="#faq23">Can I participate in the project if I am diagnosed with ...?</a></li>
                         <li><a href="#faq24">What are missing metadata?</a></li>
+						<li><a href="#faq25">How are the samples and data processed?</a></li>
```
Comment (ElDeveloper) [inline www/americangut/FAQ.psp]: @wasade seems like this is indented with tabs and all the other lines are indented with spaces, would you mind changing that?

**Comment by ElDeveloper** [comment]
Thanks @wasade

**Pull request merged**